lib/Client.mjs

```diff
@@ -12,7 +12,24 @@ class Client {
       type: 'udp4',
       reuseAddr: true
     });
+    this._bound = false;
   }
+  setBroadcast(flag, cb) {
+    if (!cb) cb = () => {};
+    if (typeof flag !== 'boolean') {
+      throw new TypeError('flag must be a boolean');
+    }
+    if (this._bound) {
+      this._sock.setBroadcast(flag);
+      cb();
+      return;
+    }
+    this._sock.bind(this.port, () => {
```

Comment:
@MylesBorins Thank you so much!
I can mostly achieve what I want to do.
In my environment, only this part didn’t work properly. It works if I remove the port specification.



```
this._sock.bind(() => {
```

Reply:
If you remove the specified port in the call to bind then node will just pick a random port, that really isn't desirable functionality. Are you getting an error?

Reply:
I understand this specification. No problem!

Because I was testing sending and receiving on the same PC, the port number was duplicated and the callback was not called.

Reply:
@narikei do you have an example script of what you are trying to do that I can play with? I'm not running into the same problem regarding duplicated port number and binding between server + client in my tests.